cognite/client/data_classes/_base.py

```diff
@@ -140,6 +141,15 @@ def dump(self, camel_case: bool = True) -> dict[str, Any]:
         """
         return basic_instance_dump(self, camel_case=camel_case)
 
+    def dump_yaml(self, filepath: str | Path) -> None:
+        """Dump the instance into a YAML file.
+
+        Args:
+            filepath (str | Path): Path to the YAML file.
+        """
+        yaml = local_import("yaml")
+        Path(filepath).write_text(yaml.dump(self.dump(camel_case=True), sort_keys=False))
```

Comment:
I assume this takes care of closing the file handle after writing?

Reply:
Yes it does.